packages/contracts-bedrock/.gitignore

```diff
@@ -1,22 +1,9 @@
 artifacts
-forge-artifacts
-cache
 typechain
 coverage.out
-.deps
 broadcast
-genesis.json
-src/contract-artifacts.ts
-tmp-artifacts
-deployments/mainnet-forked
 deploy-config/mainnet-forked.json
```

Comment:
hmm i feel like we still need some of this stuff

- forge-artifacts
- cache
- resource-metering.csv

the deployments subdirs/files